### PR TITLE
add redis eval,evalSha,script command support

### PR DESCRIPTION
**Pull request by kimkit**

### CODE REVIEW

File: swoole_redis_coro.c [hunk 1]
```diff
@@ -3457,6 +3463,164 @@ static PHP_METHOD(swoole_redis_coro, exec)
...
+    SW_REDIS_COMMAND_CHECK
+    int i = 0;
+    size_t *argvlen = emalloc(sizeof(size_t) * (params_num + 3));
+    char **argv = emalloc(sizeof(char *) * (params_num + 3));
+
+    SW_REDIS_COMMAND_ARGV_FILL("EVAL", 4)
+    SW_REDIS_COMMAND_ARGV_FILL(script, script_len)
+
+    char keys_num_str[32] = {0};
+    sprintf(keys_num_str, "%ld", params_num);
```
Comment (yangxikun) [inline swoole_redis_coro.c]: params_num应该为keys_num

**Comment by kimkit** [comment]
@yangxikun thanks.

**Comment by yangxikun** [comment]
eval、evalSha的传参形式希望参考：[phpredis::eval](https://github.com/phpredis/phpredis#eval)、[phpredis::evalSha](https://github.com/phpredis/phpredis#evalsha)

**Comment by kimkit** [comment]
@yangxikun I have make args, num_keys param optional.

**Pull request merged**